### PR TITLE
Update Emerald (Harvard) and add JMP dependent style

### PR DESCRIPTION
**Pull request by bwiernik**
Update Emerald (Harvard) to current specifications and add a dependent
style for the Journal of Managerial Psychology

### CODE REVIEW

**Comment by bwiernik** [comment]
Emerald is one publisher where "Reports" and "Working Papers" have different formatting requirements, which messes with the standard practice for Zotero of storing working papers as Report. I formatted the CSL report type to correspond to the format for government documents and the like.

**Comment by adam3smith** [comment]
No objections content-wise, but would you mind running this through http://formatter.citationstyles.org/ or something equivalent to fix the indenting (2 spaces per level is what we use)

**Comment by bwiernik** [comment]
Ah, sorry. Switched text editors and forgot to change the indenting settings.

**Comment by adam3smith** [comment]
good by me. Getting tired, so leaving this for @rmzelle to merge or doing it tomorrow so I don't overlook anything.

File: dependent/journal-of-managerial-psychology.csl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>Journal of Managerial Psychology</title>
+    <id>http://www.zotero.org/styles/journal-of-managerial-psychology</id>
+    <link href="http://www.zotero.org/styles/journal-of-managerial-psychology" rel="self"/>
+    <link href="http://www.zotero.org/styles/emerald-harvard" rel="independent-parent"/>
+    <category citation-format="author-date"/>
```
Comment (zuphilip) [inline dependent/journal-of-managerial-psychology.csl]: I suggest to add here also the field category, i.e.
```
<category field="psychology"/>
```

**Comment by adam3smith** [comment]
OK, I've got one more question: what's the story of the DOI format? I can't find any reference to DOIs anywhere, neither with doi.org/ nor with doi: Did we check existing articles?

**Comment by bwiernik** [comment]
DOIs and URLs are only used for electronic-only sources, and I've had a hard time finding any existing articles with references to either. Emerald's guidelines only refer to URLs and they don't seem to acknowledge that DOIs exist at all. I figured that a URL-formatted DOI would be preferable to a more fragile URL if it exists.

**Comment by bwiernik** [comment]
On the article landing page for the Journal of Managerial Psychology, though, they do display the article DOI formatted as a URL.

**Comment by bwiernik** [comment]
Would you prefer I do something else with the DOIs?

**Comment by adam3smith** [comment]
nono, that was exactly the answer I was looking for. I just had a super busy couple of days and haven't gotten to this yet.

**Comment by bwiernik** [comment]
No worries. Did you want me to squash the commits?

**Comment by adam3smith** [comment]
no need, we just do that for Zotero translators or for excessive commit
histories here.

On Wed, Dec 9, 2015 at 6:25 PM, bwiernik <notifications@github.com> wrote:

> No worries. Did you want me to squash the commits?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1807#issuecomment-163446038>
> .
>



-- 
Sebastian Karcher, PhD
www.sebastiankarcher.com

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**